recipes/soci/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(conanSociTest CXX)
+
+option(ENABLE_CONAN "Enable conan" True)
```

Comment:
```suggestion
```

Keep the test package as simple as possible. Avoid dual test, try to use the same code for both libraries.